src/_gettsim_tests/test_data/erziehungsgeld/2005/hh_id_4.yaml

```diff
@@ -0,0 +1,88 @@
+---
+info:
+  note: ''
+  source: own calculations
+inputs:
+  provided:
+    hh_id:
+      - 4
+      - 4
+      - 4
+    tu_id:
+      - 4
+      - 4
+      - 4
+    p_id:
+      - 8
+      - 9
+      - 10
+    alter:
+      - 40
+      - 40
+      - 0
+    arbeitsstunden_w:
+      - 29.0
+      - 40.0
+      - 0.0
+    in_ausbildung:
+      - false
+      - false
+      - false
+    bruttolohn_m:
+      - 1400.0
+      - 3000.0
+      - 0.0
+    bruttolohn_vorj_m:
+      - 1400.0
+      - 3000.0
+      - 0.0
+    geburtstag:
+      - 1
+      - 1
+      - 1
+    geburtsmonat:
+      - 1
+      - 1
+      - 8
+    hat_kinder:
+      - true
+      - true
+      - false
+    kind:
+      - false
+      - false
+      - true
+    alleinerz:
+      - false
+      - false
+      - false
+    geburtsjahr:
+      - 1965
+      - 1965
+      - 2004
+    budget_erz_geld:
+      - false
+      - false
+      - true
+    inanspruchn_erzgeld:
+      - true
```

Comment:
We shouldn't have inanspr true and then there is none. This doesn't make sense

Reply:
That's true, however, I then wonder why we even check for eligibility for erziehungsgeld if "inanspruchn_erzgeld" is only true if money is also paid out.